### PR TITLE
Account for connecting to older APIs

### PR DESCRIPTION
**Pull request by cmc333333**
This is a problem if -site and -core are out of sync.

### CODE REVIEW

File: regulations/generator/api_reader.py [hunk 1]
```diff
@@ -85,9 +85,16 @@ def layer(self, layer_name, doc_type, label_id, version=None):
...
             doc_id = '{}/{}'.format(version, root)
-        return self._get(
+        result = self._get(
             ('layer', layer_name, doc_type, root, str(version)),
             'layer/{}/{}/{}'.format(layer_name, doc_type, doc_id))
+        # To remove - also try the old format for CFR layers; the API may not
+        # have been updated
+        if result is None and doc_type == 'cfr':
+            result = self._get(
+                ('layer', layer_name, 'old', layer_name, root, str(version)),
```
Comment (jmcarp) [inline regulations/generator/api_reader.py]: The "old" here is for keeping these values from cluttering the cache?

Comment (cmc333333) [inline regulations/generator/api_reader.py]: It was just a placeholder, but thinking about it again, there's no reason to not use the same key. Will update.

**Comment by jmcarp** [comment]
Left a quick question on the diffs, but this looks right. By the way, this reminds me to nag @cmc333333 about versioning these repos and installing from the PyPI instead of github at some point.

**Comment by cmc333333** [comment]
@jmcarp versioning the repo wouldn't have helped here (we had an up-to-date -site querying a non-up-to-date -core over HTTP), but I agree with the sentiment.

Updated!

**Pull request merged**